libs/ui/src/button_list.tsx

```diff
@@ -10,6 +10,7 @@ export const ButtonList = styled.p`
   }
   & > button {
     margin-bottom: 0.5rem;
+    width: 100%;
```

Comment:
Do you have good reason to suspect this won't have unintended impacts in other places?

Reply:
Yeah, good question - I noticed all the usage instances of `ButtonList` are relying on explicitly setting the (now-deprecated) `fullWidth` prop of the `Button` component (which just applies `width: 100%` in legacy mode). `fullWidth` is ignored with the new themes, so  decided to move that logic up into `ButtonList`, since it semantically fits better there.